src/types.d.ts

```diff
@@ -1,11 +1,24 @@
-/* eslint-disable @typescript-eslint/no-redundant-type-constituents */
-import type Prism from './core/classes/prism';
-export type * from './classes/grammar';
-export type * from './classes/language';
-export type * from './classes/registry';
-export type * from './classes/plugin-registry';
-export type * from './classes/language-registry';
-export type * from './util/types';
```

Comment:
Unfortunately, this doesn't work with types. Even removing `type` doesn't solve the issue. We need to manually re-export the needed types.